docs/cf-cli-plugin.md

```diff
@@ -0,0 +1,210 @@
+# Plug-in for the “Cloud Foundry CLI”
+“Application Autoscaler” offers a plug-in for the “Cloud Foundry CLI”. With it users can attach or detach scaling-policies to their apps. They can view them and view the scaling-history. And they can view the metrics send by their apps.
```

Comment:
As they can also view built-in metrics I would phrase this sentence a bit more general 
```suggestion
“Application Autoscaler” offers a plug-in for the “Cloud Foundry CLI”. With it users can attach or detach scaling-policies to their apps. They can view them and view the scaling-history. And they can view the metrics the scaling policies are based on.
```